crates/file/src/blob.rs

```diff
@@ -147,46 +149,67 @@ pub struct File {
 impl File {
     /// Create a new `File` with the given name and contents.
     ///
-    /// `contents` can be from one of `&str`, `&[u8]`, or others.
+    /// `contents` can be `&str`, `&[u8]`, or `js_sys::ArrayBuffer`.
     pub fn new<T>(name: &str, contents: T) -> File
     where
         T: BlobContents,
     {
         Self::new_(name, contents.into_jsvalue(), None, None)
     }
 
-    /// Like `File::new`, but allows customizing the last modified time and the MIME type (also
-    /// known as *content type* or *media type*)
+    /// Like `File::new`, but allows customizing the MIME type (also
+    /// known as *content type* or *media type*), and the last modified time.
+    ///
+    /// `std::time::SystemTime` is a low level type, use a crate like
+    /// [`chrono`](https://docs.rs/chrono/0.4.10/chrono/) to work with a more user-friendly
+    /// representation of time.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use chrono::prelude::*;
+    /// use gloo_file::File;
+    ///
+    /// // Just create a dummy `gloo::file::File` for demonstration purposes.
+    /// let example_file = File::new_with_options(
+    ///     "motivation.txt",
+    ///     "live your best life",
+    ///     Some("text/plain"),
+    ///     Some(Utc::now().into())
+    /// );
+    /// assert_eq!(example_file.name(), "motivation.txt".into());
+    /// assert_eq!(example_file.raw_mime_type(), "text/plain".into());
+    /// ```
     pub fn new_with_options<T>(
         name: &str,
         contents: T,
         mime_type: Option<&str>,
-        last_modified_since_epoch: Option<Duration>,
+        last_modified: Option<SystemTime>,
```

Comment:
```suggestion
        last_modified_time: Option<SystemTime>,
```

Reply:
`last_modified` would match the underlying webidl, but I agree with you that adding `_time` is more readable.

Reply:
With gloo we don't try to match 1-to-1, that's what web-sys is for.